content_blocker_update.sh

```diff
@@ -12,7 +12,6 @@ npm install
 npm run build
 
 # Clone shavar prod list
-cd firefox-ios
 rm -rf shavar-prod-lists && git clone https://github.com/mozilla-services/shavar-prod-lists.git && git -C shavar-prod-lists checkout $SHAVAR_COMMIT_HASH
 
-(cd Client/ContentBlocker/ContentBlockerGenerator && swift run)
+(cd BrowserKit && swift run)
```

Comment:
Won't this build and run every target in BrowserKit? I thought we would need to specify only to execute the `ContentBlockingGenerator` target?

Reply:
So from what I understood, since we [only have an executable](https://github.com/mozilla-mobile/firefox-ios/blob/ed9b32d9b54ae91a702864c85ff6cce4fe4098b3/BrowserKit/Package.swift#L36-L38) for the ContentBlockingGenerator in Package.swift, this works 🤔 I'm not sure how to point otherwise to this specific executable from the command line

Reply:
My mistake, I must've been looking at the old target list and got mixed-up. Thanks for clarifying, seems like this is fine for now and we can always update it later if needed.